tests/test_model_methods.py

```diff
@@ -309,6 +309,28 @@ def test_setup_ksathorfrac(tmpdir, example_wflow_model):
     assert int(mean_val * 100) == 19991
 
 
+def test_setup_ksatver_vegetation(tmpdir, example_wflow_model):
+    # Read the modeldata
+    model = "wflow"
+    example_wflow_model.read()
+
+    # Build the map
+    example_wflow_model.setup_ksatver_vegetation(
+        soil_fn="soilgrids",
+    )
+
+    # Set the output directory
+    destination = str(tmpdir.join(model))
+    example_wflow_model.set_root(destination, mode="w")
```

Comment:
I think these ones are also optional? Only if you want to write the model files for testing but if it is just about calling the setup method and testing the values like you do, I think you can skip the set_root.